.eslintrc.js

```diff
@@ -67,7 +67,28 @@ module.exports = {
   },
   settings: {
     'import/resolver': {
-      typescript: {},
+      typescript: {
+        // Specify the path to the TypeScript configuration file
+        project: './tsconfig.packages.json',
+        // Define additional paths for module resolution to match TypeScript paths
+        paths: ['apps/*', 'packages/*'],
```

Comment:
The 'paths' option allows ESLint to understand module aliases defined in TypeScript. Here, 'apps/*' and 'packages/*' correspond to the directories where your workspace packages reside.